src/main.c

```diff
@@ -3466,6 +3468,7 @@ static int openDatabase(
   /* Open the backend database driver */
   rc = sqlite3BtreeOpen(db->pVfs, db->pWalMethods, zOpen, db, &db->aDb[0].pBt, 0,
                         flags | SQLITE_OPEN_MAIN_DB);
+
```

Comment:
let's drop the extra whitespace. It may seem nitpicky, but extra lines like that can really ruin git diffs when merging with upstream SQLite